packages/react-components/react-tags/src/components/Tag/Tag.types.ts

```diff
@@ -45,6 +46,7 @@ export type TagProps = ComponentProps<Partial<TagSlots>> & {
   // checked?: boolean;
   disabled?: boolean;
   dismissible?: boolean;
```

Comment:
`dsmissButton` should be typed with `ARIAButtonProps` in this case, no?!

Reply:
I have `dismissButton?: Slot<'button'>;` is it okay? or does it need ARIAButtonProps wrapper?

Reply:
After chat, updated it to keep `dismissButton` slot typing, and use simple resolveShorthand instead of useAriaButton :)